### ISSUE TITLE
Custom filename not work at all !

### ISSUE THREAD

**Issue by VitaliyAT** [jely2002/youtube-dl-gui#308]
Any variations of custom filenaming is not working at all. Would you fix it?
Also you can find lot of unclosed bug report about some independent filename options - but I should say that any of them not working.

**Comment by Amalik0301** [comment]
Yeah...I've been trying to customize the titles too, and almost none of the youtube-dl output names work. I found that things like the dislikes and likes, view count, and other details work, but can't access upload date or other stuff... maybe it just isn't accessed from the API as metadata.. idk

**Comment by jely2002** [comment]
The issue with playlist_index is known yeah. In v3 the custom filename template should be fixed though. Closing this a such.
